ProcessSmarterTestPackage/ProcessSmarterTestPackage.csproj

```diff
@@ -30,77 +30,6 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
-  <ItemGroup>
```

Comment:
With the latest tools, explicitly listing System and project source is not necessary.